src/test/java/org/takes/facets/auth/PsByFlagTest.java

```diff
@@ -23,34 +23,111 @@
  */
 package org.takes.facets.auth;
 
+import com.google.common.collect.ImmutableMap;
 import java.io.IOException;
+import nl.jqno.equalsverifier.EqualsVerifier;
+import nl.jqno.equalsverifier.Warning;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
+import org.junit.Before;
 import org.junit.Test;
+import org.mockito.Mockito;
+import org.takes.Response;
 import org.takes.rq.RqFake;
+import org.takes.rs.RsWithBody;
+import org.takes.rs.RsWithStatus;
+import org.takes.rs.RsWithType;
 
 /**
  * Test case for {@link PsByFlag}.
+ *
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 0.10
  */
 public final class PsByFlagTest {
+    /**
```

Comment:
Why this file is in the changeset??
Looks like you've started a branch not from the master but from the another branch